auction/program/src/processor.rs

```diff
@@ -414,10 +433,10 @@ impl AuctionData {
             gap_val,
             minimum,
             instant_sale_price,
-            &mut self.state,
```

Comment:
Were these removed because they wernt being used?

Reply:
Yes, they were also removed from the function signature